pkgs/top-level/all-packages.nix

```diff
@@ -25213,6 +25213,14 @@ with pkgs;
   gpx = callPackage ../applications/misc/gpx { };
 
   gqrx = callPackage ../applications/radio/gqrx { };
+  gqrx-portaudio = callPackage ../applications/radio/gqrx {
+    portaudioSupport = true;
+    pulseaudioSupport = false;
+  };
+  gqrx-gr-audio = callPackage ../applications/radio/gqrx {
+    portaudioSupport = false;
+    pulseaudioSupport = false;
+  };
```

Comment:
Newbie here, why does gqrx-gr-audio not need any audio support? Is it simply unused functionality under GnuRadio? Does it cause hanging when using pipewire?

Reply:
gr-audio uses Gnuradio's audio backend. It still doesn't work for me on Pipewire. :-(